collated/main/A0141006B.md

```diff
@@ -0,0 +1,1280 @@
+	
+	@Override
+	public void setNode(Node node) {
+		helpCardScrollPane = (ScrollPane) node;	
+	}
+	
+	@Override
+	public String getFxmlPath() {		
+		return FXML;
+	}
+	
+	public ScrollPane getHelpCardPane() {
+		ScrollPane helpCardScrollPane = new ScrollPane();
+		AnchorPane testPane = new AnchorPane();
+		VBox helpCardPane = new VBox();			
+		FxViewUtil.applyFullWidth(helpCardScrollPane);		
+		helpCardScrollPane.setPrefViewportWidth(600);
+		helpCardScrollPane.setPrefViewportHeight(600);
+		setHelpCards();		
+		helpCardPane.getChildren().addAll(
+				addCard, blockCard, clearCard, deleteCard, doneCard, editCard, 
+				findCard, helpCard, listCard, selectCard, storageCard, exitCard);
+		testPane.getChildren().add(helpCardPane);
+		helpCardScrollPane.setContent(testPane);
+		return helpCardScrollPane;
+	}
+
+	private void setHelpCards() {
+		setAddInfo();
+		setBlockInfo();
+		setClearInfo();
+		setDeleteInfo();
+		setDoneInfo();
+		setEditInfo();
+		setFindInfo();
+		setHelpInfo();
+		setListInfo();
+		setSelectInfo();
+		setStorageInfo();
+		setExitInfo();
+	}
+	
+	private void setExitInfo() {
+		exitCard = new VBox();
+		setPadding(exitCard);
+		exitNameLabel = new Label();
+		exitNameLabel.setText("12. " + ExitCommand.COMMAND_WORD);
+		exitDescriptionLabel = new Label();
+		exitDescriptionLabel.setText(ExitCommand.MESSAGE_USAGE);
+		exitCard.getChildren().addAll(exitNameLabel, exitDescriptionLabel);
+	}
+	
+	private void setStorageInfo() {
+		storageCard = new VBox();
+		setPadding(storageCard);
+		storageNameLabel = new Label();
+		storageNameLabel.setText("11. " + StorageCommand.COMMAND_WORD);
+		storageDescriptionLabel = new Label();
+		storageDescriptionLabel.setText(StorageCommand.MESSAGE_USAGE);
+		storageCard.getChildren().addAll(storageNameLabel, storageDescriptionLabel);
+	}
+	private void setSelectInfo() {
+		selectCard = new VBox();
+		setPadding(selectCard);
+		selectNameLabel = new Label();
+		selectNameLabel.setText("10. " + SelectCommand.COMMAND_WORD);
+		selectDescriptionLabel = new Label();
+		selectDescriptionLabel.setText(SelectCommand.MESSAGE_USAGE);
+		selectCard.getChildren().addAll(selectNameLabel, selectDescriptionLabel);
+	}	
+	private void setListInfo() {
+		listCard = new VBox();
+		setPadding(listCard);
+		listNameLabel = new Label();
+		listNameLabel.setText("9. " + ListCommand.COMMAND_WORD);
+		listDescriptionLabel = new Label();
+		listDescriptionLabel.setText(ListCommand.MESSAGE_USAGE);
+		listCard.getChildren().addAll(listNameLabel, listDescriptionLabel);	
+	}
+	private void setHelpInfo() {
+		helpCard = new VBox();
+		setPadding(helpCard);
+		helpNameLabel = new Label();
+		helpNameLabel.setText("8. " + HelpCommand.COMMAND_WORD);
+		helpDescriptionLabel = new Label();
+		helpDescriptionLabel.setText(HelpCommand.MESSAGE_USAGE);
+		helpCard.getChildren().addAll(helpNameLabel, helpDescriptionLabel);
+	}
+	private void setFindInfo() {
+		findCard = new VBox();
+		setPadding(findCard);
```

Comment:
heyyyyyyyyyy.................couldn't u jux declare an array of "Label" here ....all these functions are unnecessary ~ Extract similar code blocks as utility methods.

didn't i mention don't try to game the system  ~